docs/features/software-catalog/descriptor-format.md

```diff
@@ -43,7 +43,7 @@ software catalog API.
   "kind": "Component",
   "metadata": {
     "annotations": {
-      "backstage.io/managed-by-location": "file:/tmp/component-info.yaml",
+      "backstage.io/managed-by-location": "file:/tmp/catalog-info.yaml",
```

Comment:
I haven't tried this example, but I'll be honest I would have expected this to be in the format `file://` (double slash)?

Reply:
No it's not a `file://` url schema but a backstage type `file:` location. However I'm not sure if they are still supported or if they are now part of the type `url:`